karma.conf.ts

```diff
@@ -34,7 +34,9 @@ const testRecursivePath = "test/visualTest.ts";
 const srcOriginalRecursivePath = "src/**/*.ts";
 const coverageFolder = "coverage";
 
-process.env.CHROME_BIN = require("playwright").chromium.executablePath();
+process.env.CHROME_BIN = require("puppeteer").executablePath();
+
+//import { Config, ConfigOptions } from "karma";
```

Comment:
Why is it commented?

Reply:
That was a left over part of the code from when I forked the original repo. I missed it when removing commented out code. It has been removed now: [81189d4](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/81189d443a2bda4f576855676a9396fd4aace44a)